wsclient/src/main/java/org/hpccsystems/ws/client/HPCCWsDFUClient.java

```diff
@@ -49,12 +51,11 @@
  */
 public class HPCCWsDFUClient extends DataSingleton
 {
-    private static URL                  originalURL;
+    private static URL originalURL;
 
     public static URL getOriginalURL() throws MalformedURLException
     {
-        if (originalURL == null)
-            originalURL = new URL(getOriginalWSDLURL());
+        if (originalURL == null) originalURL = new URL(getOriginalWSDLURL());
```

Comment:
I was surprised at the format changes. I ran the formatter after I made my updates to the file using the HPCC-JAPI template, expecting the only changes to by my additions; apparently there were prior updates to the file made without being formatted. Sorry about that one